docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -0,0 +1,106 @@
+- **Query sharding cardinality estimation.** You can enable it through the advanced CLI configuration flag `-query-frontend.query-sharding-target-series-per-shard`, allows query sharding to take into account cardinality of similar requests executed previously when computing the maximum number of shards to use. If you want to try it out, we recommend starting with a value of `2500`.
+- **Query expression size limit**. You can limit the size in bytes of the queries allowed to be processed through the CLI configuration flag `-query-frontend.max-query-expression-size-bytes`.
+- **Peer discovery / tenant sharding for overrides exporters**. You can enable it through the CLI configuration flag `-overrides-exporter.ring.enabled`.
+- **Overrides exporter enabled metrics selection**. You can select which metrics the overrides exporter should export through the CLI configuration flag `-overrides-exporter.enabled-metrics`.
+- **Per-tenant results cache TTL**.
+
+## Experimental features
+
+Grafana Mimir 2.10 includes new features that are considered as experimental and disabled by default. Please use them with caution and report any issues you may encounter:
+
+- **Support for ingesting exponential histograms in OpenTelemetry format**. The exponential histograms that are over the native histogram scale limit of 8 can be downscaled to allow their ingestion.
+- **Store-gateway index-header loading improvements**, which include the ability to persist the sparse index-header to disk instead of reconstructing it on every restart (`-blocks-storage.bucket-store.index-header-sparse-persistence-enabled`) and an option to limit the number of concurrent index-header loads when lazy-loading (`-blocks-storage.bucket-store.index-header-lazy-loading-concurrency`).
+- **Option to allow queriers to reduce pressure on ingesters** by initially querying only the minimum set of ingesters required to reach quorum. (`-querier.minimize-ingester-requests`).
+- **Early TSDB Head compaction in the ingesters** to reduce in-memory series when a certain threshold is reached. Useful to deal with high series churning rate. (`-blocks-storage.tsdb.early-head-compaction-min-in-memory-series`).
+- **Spread-minimizing token generation algorithm** for the ingesters. This new method drastically reduces the difference in series pushed to different ingesters. Please note that [a migration process](https://github.com/grafana/mimir/issues/4736#issuecomment-1602976040) is required to switch from previous random generation algorithm, which will be detailed once the feature is declared stable.
+- **Support for chunks streaming from store-gateways to queriers** that should reduce the memory usage in those components. Can be enabled through the `-querier.prefer-streaming-chunks-from-store-gateways` option.
```

Comment:
This should only affect the queriers, less the store-gateways, since store-gateways hold the same number of chunks in memory as before, only now they send them in smaller batches.

Reply:
Changed, thanks.